### PR TITLE
[IMP] web_tour: Disable tours when Odoo installed with demo data

### PR DESCRIPTION
**Pull request by jat-odoo**
Description of the issue/feature this PR addresses:
* Task: https://www.odoo.com/web?debug#id=37026&view_type=form&model=project.task&action=327&menu_id=4720
* Pad: https://pad.odoo.com/p/r.28923badc10fcba93f89b51d4c81b198

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by msh-odoo** [comment]
LGTM

File: addons/web_tour/__manifest__.py [hunk 1]
```diff
@@ -16,6 +16,9 @@
         'views/tour_templates.xml',
         'views/tour_views.xml'
     ],
+    'demo': [
+        'data/web_tour_data.xml',
```
Comment (tivisse) [inline addons/web_tour/__manifest__.py]: According to the guidelines, the file should be name ``web_tour_demo.xml``

File: addons/web_tour/static/src/js/tour_disable.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+"use strict";
+
+var TourManager = require('web_tour.TourManager');
+
+TourManager.include({
+    /**
+     * Disable tours if Odoo installed with demo data.
+     *
+     * @override
+     * @private
```
Comment (tivisse) [inline addons/web_tour/static/src/js/tour_disable.js]: According to the guidelines ``@override`` allows to avoid repeating the params and returns declarations.

File: addons/web_tour/static/src/js/tour_disable.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+     * @param {boolean} do_update
+     * @param {Object} tour
+     * @param {string} name
+     * @returns {Deferred}
+     */
+    _register: function (do_update, tour, name) {
+        // Consuming tours which are not run by test case
+        if (!this.running_tour) {
```
Comment (tivisse) [inline addons/web_tour/static/src/js/tour_disable.js]: I think that this condition is always true, why did you add it ?

**Comment by tivisse** [comment]
Landed in master at https://github.com/odoo/odoo/commit/b7eee16a585667e50aac0a50d0abb2c35660df82
